bases/br_sp_gov_orcamento/despesas/publish.sql

```diff
@@ -0,0 +1,39 @@
+/*
+
+Query para publicar a tabela.
+
+Esse é o lugar para:
+    - modificar nomes, ordem e tipos de colunas
+    - dar join com outras tabelas
+    - criar colunas extras (e.g. logs, proporções, etc.)
+
+Qualquer coluna definida aqui deve também existir em `table_config.yaml`.
+
+# Além disso, sinta-se à vontade para alterar alguns nomes obscuros
+# para algo um pouco mais explícito.
+
+TIPOS:
+    - Para modificar tipos de colunas, basta substituir STRING por outro tipo válido.
+    - Exemplo: `SAFE_CAST(column_name AS NUMERIC) column_name`
+    - Mais detalhes: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
+
+*/
+SELECT 
+SAFE_CAST(orgao AS STRING) orgao,
```

Comment:
To pensando se não vale a pena adicionar uma coluna com o id_municipio e outra com o id_estado.

Assim, aplicações futuras para saber que base cruzar com o que poderão usar essas colunas.

A implementação é simples:
```sql
XXXXXX as id_municipio,
YYYYYY as id_estado
```

o que você acha @rdahis  e @polvoazul

Reply:
Com certeza, vamos adicionar. Teria que dar um merge com o diretório né? É pelo publish.sql?

Reply:
Faz sentido, no caso do id_estado seria SP = 35. Já pro id_municipio ficaria XXXXXX?

Reply:
Não, é só escrever no publish.sql mesmo

```
123454 as id_municipio,
35 as id_estado
```

Reply:
@JoaoCarabetta  quando do tb.update() ele n tá dando update no publish.sql